doc/dev/developer_guide/testing_integration_tests/index.rst

```diff
@@ -0,0 +1,15 @@
+=======================
+Teuthology User Guide
+=======================
+
+.. rubric:: Contents
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   Introduction <tests-integration-testing-teuthology-intro>
+   Workflow <tests-integration-testing-teuthology-workflow>
+   Debugging Tips <tests-integration-testing-teuthology-debugging-tips>
```

Comment:
did you intend for this to look like https://ceph--37949.org.readthedocs.build/en/37949/dev/developer_guide/testing_integration_tests/? Seems like Debugging tips should enough here